Source/SwiftyDropbox/Shared/Handwritten/DropboxTransportClient.swift

```diff
@@ -67,15 +67,17 @@ public class DropboxTransportClientImpl: DropboxTransportClientInternal {
         appSecret: String,
         baseHosts: BaseHosts = .default,
         userAgent: String?,
-        authChallengeHandler: @escaping AuthChallenge.Handler
+        authChallengeHandler: @escaping AuthChallenge.Handler,
+        headersForRouteHost: HeadersForRouteRequest? = nil
```

Comment:
There's a few other parameters in the accessTokenProvider-based initializers. Are there any other ones we want to include here?